### ISSUE TITLE
A11y: No announcement of the current quantity after adding/removing a product to/from the wish list

### ISSUE THREAD

**Issue by 73nici** [shopware/platform#7084]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

After adding / removing a product from the watch list, the change is not communicated to screen reader users. What is particularly critical here is that the display of the number of products visible to everyone does not adjust despite successful deletion, so that screen reader users must assume that the deletion process has failed and blind users are not informed that the deletion process was successful.

*_Expected behaviour:_*

The aria-label attribute must be removed from the ‘Wishlist’ link.
Then set a role=status to announce the changes. If this cannot be added, a (possibly visually hidden) text with a role=‘status’ HTML attribute can also be added elsewhere; the number of products, e.g. ‘3 products’, must be adjusted after successful deletion or addition.

```html
<a>  <span class="icon icon-heart">...</span>      <span role="status">    <span>Merkzettel</span>    <span>1</span><span>Produkte</span>  </span></a>
```

![Image](https://github.com/user-attachments/assets/6a0d1fb7-078e-4f33-99c4-3a76536b12b1)

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag:
- add a product to the wishlistd

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40840**



[created from NEXT-40840, comment 541783]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40840, comment 542004]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40840, comment 542159]

**Comment by blacksheep--** [comment]
To add from #7107, because it seems not to be covered here: this applies to the cart icon also.

**Comment by bschulzebaek** [comment]
While it's a similar topic, we moved the cart behavior to a new issue, since it's a tricky element for screen readers and requires a bit more consideration to be done right -> #9229
